{"name": "test_login_error", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('get_data_2', <FixtureRequest for <Function test_login_error>>)", "trace": "file C:\\Users\\prach\\PycharmProjects\\PythonTesting\\SeleniumAutomationPractise\\tests\\test_login.py, line 23\n      def test_login_error(self, setup,get_data_2):\nE       fixture 'get_data_2' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, extra, get_data, get_data2, include_metadata_in_junit_xml, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, setup, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Users\\prach\\PycharmProjects\\PythonTesting\\SeleniumAutomationPractise\\tests\\test_login.py:23"}, "start": 1653661858181, "stop": 1653661858181, "uuid": "16896997-4893-4cb8-833f-cfb7d6c6f557", "historyId": "d0a08302675d12a6513453a67870e4cf", "testCaseId": "0d76f03330ef19c969af1bc2c1c2d1ce", "fullName": "test_login.TestLogin#test_login_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-5NPQ059D"}, {"name": "thread", "value": "17224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}